debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresOffsetContext.java

```diff
@@ -232,7 +232,7 @@ public static PostgresOffsetContext initialContext(PostgresConnectorConfig conne
         try {
             LOGGER.info("Creating initial offset context");
             final Lsn lsn = Lsn.valueOf(jdbcConnection.currentXLogLocation());
-            final long txId = jdbcConnection.currentTransactionId().longValue();
+            final Long txId = jdbcConnection.currentTransactionId();
```

Comment:
Code elsewhere was prepared for the TX id to be null already, so I think the change just highlighted this incorrect usage here.